manifests/install.pp

```diff
@@ -0,0 +1,38 @@
+# == Class: gdm::install
+#
+# This class installs gdm dependencies.
+#
+# == Authors
+#
+# * Trevor Vaughan <tvaughan@onyxpoint.com>
+# * Nick Markowski <nmarkowski@keywcorp.com>
+#
+class gdm::install {
+  assert_private()
+
+  # GNOME Display Manager
+  package { 'gdm': ensure => 'latest' }
+
+  # Basic XOrg Stuff
+  package { $::gdm::x_package_list :
+    ensure => 'latest',
+  }
+
+  # Fonts
+  package {
```

Comment:
This list should be a parameter and should probably use data in modules or params.pp